src/client/src/routes/OrderTicketRoute/MediaPlayer.tsx

```diff
@@ -18,7 +18,7 @@ export interface State {
     position: number
     pausedAt: number
   }
-  source?: AudioBufferSourceNode
+  source?: AudioBufferSourceNode | null
```

Comment:
`AudioBufferSourceNode | null | undefined`? 😥 Any way to unify the `null | undefined`, or are they used for different scenarios?

Reply:
Good shout. Removing the undefined. Thanks :)